### PR TITLE
Final build for SCP:CB 1.3.10 (It's ready for release on July 29th)

### PR DESCRIPTION
**Pull request by ENDSHN**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,4 @@ Saves/*
 error_log_*
 .vscode
 updates/
-Thumbs.db
```
Comment (Regalis11) [inline .gitignore]: Why was this removed? I don't think the thumb files should be included in the repo.

Comment (ENDSHN) [reply .gitignore]: They have to be, otherwise the "jorge" command is broken.

Comment (juanjp600) [reply .gitignore]: who cares lol

Comment (Bluscream) [reply .gitignore]: What does that command even do?

Comment (Oleg720) [reply .gitignore]: Bluscream, Try and learn XD

Comment (Regalis11) [reply .gitignore]: If we really do have to keep an obscure inside joke like that in the game, there are better ways to do it than abusing thumbs files.

Comment (ENDSHN) [reply .gitignore]: Well, if you want you can just readd the Thumbs.db file in the gitignore @Regalis11 as I personally don't care about that too much.
